test/medium-client-integration-tests.js

```diff
@@ -0,0 +1,120 @@
+var config = require("./load-config")
+var medium = require('../')
+var should = require('should')
+var testHelper = require('./test-helper')
+
+if (config.accessToken) {
+
+  var validateString = function (str, opt_expected) {
+    str.should.be.String().and.not.be.empty()
+    if (opt_expected) {
+      str.should.equal(opt_expected)
+    }
+  }
+
+  var validateArray = function (arr, opt_expected) {
+    arr.should.be.Array().and.not.be.empty()
+    if (opt_expected) {
+      arr.should.deepEqual(opt_expected)
+    }
+  }
+
+  describe("MediumClient", function () {
```

Comment:
These unit tests are very basic and just make sure proper options are enforced for various calls. I could look into mocking http/https object and also ensuring API calls look as expected - that is only covered through integration tests for now.